src/components/QACertTestSummaryHeaderInfo/QACertTestSummaryHeaderInfo.js

```diff
@@ -50,6 +50,7 @@ export const QACertTestSummaryHeaderInfo = ({
   ///
   setUpdateRelatedTables,
 }) => {
+  console.log('sectionSelect',sectionSelect,configID)
```

Comment:
Can you remove the console.logs from this file unless you need them for something.